Cargo.toml

```diff
@@ -25,8 +24,10 @@ tokio = { version = "1.21", features = ["macros", "rt-multi-thread", "time"] }
 
 # use this if you want to be able to change both repos in the same time
 # iotics-grpc-client = { path = "../iotics-grpc-client-rs" }
-iotics-grpc-client = "4.0"
+# iotics-grpc-client = "4.0"
+iotics-grpc-client = { git = "https://github.com/Iotic-Labs/iotics-grpc-client-rs.git", branch="ei-2921" }
 
 # use this if you want to be able to change both repos in the same time
 # iotics-identity = { path = "../iotics-identity-go/ffi/rust" }
 iotics-identity = { git = "https://github.com/Iotic-Labs/iotics-identity-go.git" }
+dotenvy = "0.15.7"
```

Comment:
Should this be above, replacing the `dotenv` line?

Reply:
I used cargo remove / add and left it where it was added. I can move it up if you want though

Reply:
.. just thinking that it fits more with the layout before (3rd-party first) - but this is minor - up to you